src/backend/contact-request/contact-request.service.ts

```diff
@@ -12,3 +12,18 @@ export async function createContactRequest(contactRequest: ContactFormRequest) {
     },
   })
 }
+
+export async function findRedundantContactRequest({
```

Comment:
**nit:** redundant is zbędne - we should name it more properly, like `findExistingContactRequestForProfile`